otwcode/otwarchive
# AO3-6432 Prevent orphan_account from being warned/banned/suspended

# Pull Request Checklist

  as the *first* thing in your pull request title (e.g. `AO3-1234 Fix thing`)
  until they are reviewed and merged before creating new pull requests.

## Issue

https://otwarchive.atlassian.net/browse/AO3-6432

## Purpose

Prevents `orphan_account` from being able to be warned/banned/suspended by an admin. Note that it also assumes that since `orphan_account` can never be banned or suspended, it can also never be unbanned or unsuspended.

## Testing Instructions

- Log in as a superadmin or policy_and_abuse admin
- Go to `orphan_account` page in admin view (endpoint `/admin/users/orphan_account`)
- Suspend or ban the account

It should give an error `orphan_account cannot be banned/suspended.`

## References

N/A

## Credit

ellieyhc (she/her)
